src/shared/components/mutationTable/column/expectedAltCopies/ExpectedAltCopiesColumnFormatter.tsx

```diff
@@ -2,19 +2,19 @@ import * as React from 'react';
 import { Mutation } from 'cbioportal-ts-api-client';
```

Comment:
* adding this comment in case we want all references of facets/FACETS replaced with "ascn/ASCN" 

found references to `FACETS`